### PR TITLE
148: Proof Coding Standard - Amir

### PR DESCRIPTION
**Pull request by AmirNaghibi**
- single quote
- 2  space tab

### CODE REVIEW

File: client/src/components/Dropdown.js [hunk 1]
```diff
@@ -4,13 +4,13 @@ import '../css/NavBar.css';
 class Dropdown extends Component {
   render() {
     return (
-      <div className="dropdown">
```
Comment (DConnorFong) [inline client/src/components/Dropdown.js:7]: I think HTML convention is to have double-quotations. Javascript is fine to change to single quotations.

File: client/src/components/FilterMenu.js [hunk 2]
```diff
@@ -74,9 +74,9 @@ class FilterMenu extends Component {
 
   // source: https://github.com/hibiken/react-places-autocomplete
   renderFunc = ({ getInputProps, getSuggestionItemProps, suggestions }) => (
-    <div className="autocomplete-root">
+    <div className='autocomplete-root'>
```
Comment (DConnorFong) [inline client/src/components/FilterMenu.js:77]: @AmirNaghibi think HTML convention is to have double-quotations. Javascript is fine to change to single quotations.

Comment (DConnorFong) [reply client/src/components/FilterMenu.js:77]: @kimoantiqe

**Review by DConnorFong — CHANGES_REQUESTED** [review]

File: client/src/actions/postingsActions.js [hunk 3]
```diff
@@ -4,15 +4,15 @@ export default function fetchPostings(searchKeywords) {
...
-          'Content-Type': 'application/json',
-        },
-        body: `{"keywords": "${searchKeywords}"}`,
-      })
+      method: 'POST',
+      headers:{
+        Accept: 'application/json',
+        'Content-Type': 'application/json',
+      },
+      body: `{'keywords': '${searchKeywords}'}`,
```
Comment (cowmanjoe) [inline client/src/actions/postingsActions.js:14]: This is a JSON literal, so I don't think it would be allowed to have single quotes.

**Review by cowmanjoe — CHANGES_REQUESTED** [review]

**Pull request closed without merging**